ui/src/components/RadioConfirm/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import * as Stores from '@stores';
+
+type Props = {
+    status: boolean;
+    updateStatusTrue: () => void;
```

Comment:
a single 'onChange' handler provided via props it's more than enough, you can do different stuff based on the parameter true/false in the parent component.